src/Microsoft.Dnx.Runtime/ApplicationHostContext.cs

```diff
@@ -215,7 +224,30 @@ private static Result InitializeCore(ApplicationHostContext context)
             var lockFilePresent = context.LockFile != null;
             context.LockFile = null; // LockFile is no longer needed
 
-            return new Result(libraries, lockFilePresent, validLockFile, lockFileValidationMessage);
+            string minRequiredRuntime = null;
+            if (!context.SkipRuntimeMinVersionCheck)
+            {
+                LibraryDescription abstractions = null;
+                foreach (var lib in libraries)
+                {
+                    // Ignore projects references because in that case we are compiling from sources 
+                    // and versions don't match
+                    if (lib.Type == LibraryTypes.Package &&
```

Comment:
`Type` is a string I believe. We should use `string.Equals` to compare it in order to be clear about case-sensitivity (it **is** case-sensitive :))